routes/index.js

```diff
@@ -538,7 +546,8 @@ router.put('/discover', function(req, res, next) {
     var applyFilter = function(songs2d, done) {
         if (!_.isEmpty(langFilter)) {
             songs2d = songs2d.filter((songs) => {
-                var langArray = (songs.map((song) => song.lang))
+                //get all the songs lang id, convert it to string so we can compare it later with langFilter
+                var langArray = (songs.map((song) => song.lang.toString()))
```

Comment:
just be consistent. Let's stick with the convent of explicit toString() on line 550